sale_coupon_auto_refresh/models/__init__.py

```diff
@@ -1,3 +1,4 @@
 from . import res_config_settings
 from . import res_company
 from . import sale_order
+from . import sale_order_line
```

Comment:
As `sale_order_line` is a `One2many` relation of `sale_order` the common practice is to put both class inherits in the same file